kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -431,6 +431,8 @@ def _create_kubernetes_state_prometheus_instance(self, instance):
         if 'labels_mapper' in ksm_instance and not isinstance(ksm_instance['labels_mapper'], dict):
             self.log.warning("Option labels_mapper should be a dictionary for %s", endpoint)
 
+        ksm_instance['metrics'].extend(ksm_instance.get('additional_metrics', []))
```

Comment:
IIRC there's other checks that use the `metrics` keyword for this (although might not be documented), probably good to standardise on that...